dependencies.gradle

```diff
@@ -11,48 +11,46 @@ ext {
     // publish correctly a new version of each of our libraries.
     // e.g. screenManager library, you could have as a current version
     // 1.0.0-beta1, if you want to publish a new version you will have to increase in one the
-    //screenManagerCoreVersionCode and to bump the screenManagerCoreVersionName.
+    // screenManagerCoreVersionCode and to bump the screenManagerCoreVersionName.
 
     // SurfaceDuo SDK libraries version code.
     // If you want to publish a new version, bump in one (1) the specific line(s)
     screenManagerCoreVersionCode = 4
     screenManagerDMVersionCode = 4
     screenManagerWMVersionCode = 4
     screenManagerUtilsVersionCode = 4
-    layoutsVersionCode = 7
-    fragmentsHandlerVersionCode = 4
-    bottomNavigationVersionCode = 3
-    tabsVersionCode = 3
-    recyclerviewVersionCode = 5
+    layoutsVersionCode = 8
+    fragmentsHandlerVersionCode = 5
+    bottomNavigationVersionCode = 4
+    tabsVersionCode = 4
+    recyclerviewVersionCode = 6
     inkSDKVersionCode = 6
```

Comment:
You should increase also the inkSdkVersionCode, no?

Reply:
We are not making a release of the ink sdk.